src/Chapter2.hs

```diff
@@ -858,7 +879,8 @@ and reverses it.
   function, but in this task, you need to implement it manually. No
   cheating!
 -}
-rewind = error "rewind: Not Implemented!"
+rewind l = foldl (\acc x -> x : acc) [] l
```

Comment:
That's absolutely correct solution 🥇 

Another solution could be with the accumulator and the recursive function that will do the addition at the start of the list which is instant!

You can read a bit more about the `go` pattern in here: https://kowainik.github.io/posts/haskell-mini-patterns#recursive-go